lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
Telling bindings to look in `module_root/compiled/version/platform/arch/bindings` 
would look for `v6.9.5/win32/x64/xmljs.node`

bindings does some string replacement on those parts of the try path